src/test/java/uk/ac/ebi/embl/api/validation/plan/ValidationPlanTest.java

```diff
@@ -87,16 +87,39 @@ public void testIsInValidationScope_Right() {
   }
 
   @Test
-  public void testIsInValidationScope_Many() {
-    assertTrue(
-        plan.isInValidationScope(
-            new ValidationScope[] {ValidationScope.EMBL, ValidationScope.INSDC}));
+  public void testIsInValidationScopePutffGroup() {
```

Comment:
(Same comment as the previous one) -> Please move the validation scope and group tests into a new class:

```
ValidationScopeTest
```

and test that all ValidationScope enums against isInGroup:

```
@Test
public void testValidationScopeIsInGroup() { 
assertTrue(ValidationScope.NCBI.isInGroup(ValidationScope.Group.PUTFF));
assertFalse(ValidationScope.NCBI.isInGroup(ValidationScope.Group.PIPELINE));
etc.
...
} 
```